src/components/ContinueActivityCard/ContinueActivityCard.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import GoToActivityCard from 'components/GoToActivityCard/GoToActivityCard';
+
+type Props = {
+  className?: string;
+};
+
+export default function ContinueActivityCard({ className }: Props) {
+  return (
+    <GoToActivityCard
```

Comment:
Do we really need this component? I am not so sure how you are looking at it in the future, but I think that GoToActivityCard.tsx should be enough. 
The only difference I see is the className prop, which if necessary could be solved by wrapping the component in a div and not creating extra components

Reply:
I believe it is importante to keep this component due to semantics. We may similar components, but they have a different purpose